### PR TITLE
Generate: Fix CI related to #20727 

### PR DESCRIPTION
**Pull request by gante**
# What does this PR do?

Fixes the error that showed up here: https://github.com/huggingface/transformers/actions/runs/3834635537/jobs/6527258530

Related to #20727

### CODE REVIEW

File: tests/generation/test_utils.py [hunk 1]
```diff
@@ -3037,7 +3037,7 @@ def test_eos_token_id_int_and_list_greedy_search(self):
         text = """Hello, my dog is cute and"""
         tokens = tokenizer(text, return_tensors="pt")
 
-        model = GPT2LMHeadModel.from_pretrained("hf-internal-testing/tiny-random-gpt2").to(torch_device)
```
Comment (ydshieh) [inline tests/generation/test_utils.py:3040]: I feel it's better to keep it is, but change (line above)

```
tokens = tokenizer(text, return_tensors="pt")
```
to
```
tokens = tokenizer(text, return_tensors="pt").to(torch_device)
```

WDYT?

Comment (gante) [reply tests/generation/test_utils.py:3040]: TIL that can be done :D 

Changed!

**Review by sgugger — APPROVED** [review]
Thanks for the fixes!

**Review by ydshieh — APPROVED** [review]
King of generate！ Thanks

**Pull request merged**